resources/views/cart/coupon.blade.php

```diff
@@ -10,7 +10,7 @@ class="text-sm"
             >
                 <div class="flex w-full flex-row gap-x-3">
                     <x-rapidez-ct::input
-                        class="text-ct-primary h-full"
+                        class="text-ct-primary w-60"
```

Comment:
is this going to work on mobile devices with that width?

Reply:
yes because, 
w-60 | width: 15rem; /* 240px */

w-60 is equal to 240px